src/main/Recorder.ts

```diff
@@ -484,6 +490,12 @@ export default class Recorder {
     this.audioInputDevices.forEach((device) => {
       const index = this.audioInputDevices.indexOf(device);
       const channel = this.audioInputChannels[index];
+
+      if (this.cfg.get('obsForceMono')) {
+        // value of osn.EsourceFlags.ForceMono
+        // eslint-disable-next-line no-bitwise
+        device.flags = 1 << 1;
```

Comment:
I've had this problem with other enums from OSN. I think they are naively exporting enums on module.d.ts but that doesn't work AIUI (we chatted about this recently in the dev channel on discord). 

Can we just reproduce their enum in obsEnums and use that to avoid having this slightly arcane bitwise syntax? 

```
const enum ESourceFlags {
    Unbuffered = (1 << 0),
    ForceMono = (1 << 1)
}
```

&&

```
if (this.cfg.get('obsForceMono')) {
  device.flags = ESourceFlags.ForceMono;
}
```

Downside is they might change it upstream, but I'd prefer this regardless I think.

Reply:
Ok, I agree that's a better way to handle it. [This page](https://www.typescriptlang.org/docs/handbook/enums.html#const-enum-pitfalls) explains why it's required.
> Inlining enum values is straightforward at first, but comes with subtle implications. These pitfalls pertain to ambient const enums only (basically const enums in .d.ts files) and sharing them between projects, but if you are publishing or consuming .d.ts files, these pitfalls likely apply to you, because tsc --declaration transforms .ts files into .d.ts files.

Perhaps we could let OSN know they could [configure TS](https://www.typescriptlang.org/tsconfig#preserveConstEnums) in such a way where we could use them without duplicating them.

Reply:
https://github.com/stream-labs/obs-studio-node/issues/1260